client/components/tests/AmenitiesCategory.test.jsx

```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+import { shallow } from 'enzyme';
+
+import AmenitiesCategory from '../AmenitiesCategory';
+
+describe('AmenitiesCategory', () => {
+  const wrapper = shallow(<AmenitiesCategory />);
+
+  it('should exist', () => {
+    expect(wrapper).toExist();
+  });
+});
```

Comment:
you can try using enzyme containsMatchingElement test method to test if the component is rendering as expected